sci-mathematics/calc/calc-2.12.5.3.recipe

```diff
@@ -0,0 +1,53 @@
+SUMMARY="C-style arbitrary precision calculator"
+DESCRIPTION="Calc is an interactive calculator which provides for \
+easy large numeric calculations, but which also can be easily programmed for \
+difficult or long calculations. It can accept a command line \
+argument, in which case it executes that single command and exits. \
+Otherwise, it enters interactive mode. In this mode, it accepts \
+commands one at a time, processes them, and displays the answers. \
+In the simplest case, commands are simply expressions which are evaluated."
+HOMEPAGE="http://www.isthe.com/chongo/tech/comp/calc/"
+COPYRIGHT="1999-2014  Landon Curt Noll"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="http://www.isthe.com/chongo/src/calc/calc-$portVersion.tar.bz2"
+CHECKSUM_SHA256="0fcf60917efb10aab145f1180c278a3ab63d789aad6d5e5f066a4458b5f9b692"
+PATCHES="calc$secondaryArchSuffix-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+PROVIDES="
+	calc$secondaryArchSuffix = $portVersion
+	cmd:calc$secondaryArchSuffix
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+         cmd:awk
+         cmd:cmp
+         cmd:less
+         cmd:ln
+	"
+
+BUILD()
+{
+         make -f Makefile.simple INCDIR=$includeDir \
+         BINDIR=$binDir \
+         LIBDIR=$libDir \
+         EXTRA_CFLAGS="-D_HAIKU_ -DUSE_TERMIOS -DHAVE_UID_T"  
```

Comment:
We have a Ctrl-V Ctrl-V Ctrl-Z between **`make -f Makefile.simple`** and **`NCDIR=$includeDir`**, and a Ctrl-X after **`LIBDIR=$libDir`**.

BTW, I forgot to mention that we usually add one extra tab for continuation lines:
```
	make -f Makefile.simple INCDIR=$includeDir \
		BINDIR=$binDir \
		LIBDIR=$libDir \
		EXTRA_CFLAGS="-D_HAIKU_ -DUSE_TERMIOS -DHAVE_UID_T"
```
Hmm, **`__HAIKU__`** gets defined automatically. Maybe we don't need to define **`_HAIKU_`**, unless the sources test for it.